### ISSUE TITLE
phantomjs --webdriver=127.0.0.1:8910 --proxy=... still listens on 0.0.0.0

### ISSUE THREAD

**Issue by colinmollenhour** [ariya/phantomjs#14275]
I'm actually using phantomjs as part of arachni, but the issue is it is opening ports on my server on the public interface. Based on all of the docs and `--help` it should listen on 127.0.0.1 by default but it is not.
1. Which version of PhantomJS are you using? Tip: run `phantomjs --version`.

2.1.1
1. What steps will reproduce the problem?

```
$ bin/phantomjs --webdriver=127.0.0.1:8910 --proxy=http://127.0.0.1:25000
[INFO  - 2016-05-12T00:05:39.288Z] GhostDriver - Main - running on port 8910
```
###### separate console

```
$ netstat -plunt | grep 8910
tcp        0      0 0.0.0.0:8910            0.0.0.0:*               LISTEN      26069/phantomjs

```
1. Which operating system are you using?

Ubuntu 14.04
1. Did you use binary PhantomJS or did you compile it from source?

Binary
1. Please provide any additional information below.

This seems to be addressed in GhostDriver already but the lastest binary release still exhibits the issue.
https://github.com/detro/ghostdriver/pull/414

**Comment by idg10** [comment]
Although that PR was accepted, there doesn't seem to have been a release that includes it. The latest is https://github.com/detro/ghostdriver/releases/tag/1.1.1 which seems to date back to January 2014. That fix wasn't written until December 2014, and the PR wasn't accepted until August 2015.

So I guess until @detro does another release, PhantomJS is stuck with this bug.

**Comment by raipc** [comment]
It seems like a solution has been merged to master branch in #14849. Is there a dev release of 2.5.0 planned?
